[2024-07-07T05:32:11.620+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo/run_id=manual__2024-07-07T05:12:16.530416+00:00/task_id=AutoML_PyCaret permission to 509
[2024-07-07T05:32:11.638+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo/run_id=manual__2024-07-07T05:12:16.530416+00:00 permission to 509
[2024-07-07T05:32:11.639+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo permission to 509
[2024-07-07T05:32:11.640+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs permission to 509
[2024-07-07T05:32:11.690+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo/run_id=manual__2024-07-07T05:12:16.530416+00:00/task_id=AutoML_PyCaret permission to 509
[2024-07-07T05:32:11.691+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo/run_id=manual__2024-07-07T05:12:16.530416+00:00 permission to 509
[2024-07-07T05:32:11.691+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo permission to 509
[2024-07-07T05:32:11.692+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs permission to 509
[2024-07-07T05:32:11.772+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: AutoML_workflow_demo.AutoML_PyCaret manual__2024-07-07T05:12:16.530416+00:00 [queued]>
[2024-07-07T05:32:11.815+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: AutoML_workflow_demo.AutoML_PyCaret manual__2024-07-07T05:12:16.530416+00:00 [queued]>
[2024-07-07T05:32:11.820+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-07-07T05:32:11.843+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): AutoML_PyCaret> on 2024-07-07 05:12:16.530416+00:00
[2024-07-07T05:32:11.860+0000] {standard_task_runner.py:60} INFO - Started process 5511 to run task
[2024-07-07T05:32:11.884+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'AutoML_workflow_demo', 'AutoML_PyCaret', 'manual__2024-07-07T05:12:16.530416+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/dmc_pipeline.py', '--cfg-path', '/tmp/tmph1u81e_6']
[2024-07-07T05:32:11.899+0000] {standard_task_runner.py:88} INFO - Job 31: Subtask AutoML_PyCaret
[2024-07-07T05:32:11.998+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo/run_id=manual__2024-07-07T05:12:16.530416+00:00/task_id=AutoML_PyCaret permission to 509
[2024-07-07T05:32:12.014+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo/run_id=manual__2024-07-07T05:12:16.530416+00:00 permission to 509
[2024-07-07T05:32:12.015+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=AutoML_workflow_demo permission to 509
[2024-07-07T05:32:12.016+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs permission to 509
[2024-07-07T05:32:12.017+0000] {task_command.py:423} INFO - Running <TaskInstance: AutoML_workflow_demo.AutoML_PyCaret manual__2024-07-07T05:12:16.530416+00:00 [running]> on host 549d24f0d639
[2024-07-07T05:32:12.130+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='AutoML_workflow_demo' AIRFLOW_CTX_TASK_ID='AutoML_PyCaret' AIRFLOW_CTX_EXECUTION_DATE='2024-07-07T05:12:16.530416+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-07T05:12:16.530416+00:00'
[2024-07-07T05:32:16.789+0000] {logging_mixin.py:188} INFO -                     Description                                 Value
0                    Session id                                   123
1                        Target                                Target
2                   Target type                            Multiclass
3                Target mapping  Dropout: 0, Enrolled: 1, Graduate: 2
4           Original data shape                           (76518, 37)
5        Transformed data shape                           (76518, 37)
6   Transformed train set shape                           (53562, 37)
7    Transformed test set shape                           (22956, 37)
8              Numeric features                                    36
9                    Preprocess                                  True
10              Imputation type                                simple
11           Numeric imputation                                  mean
12       Categorical imputation                                  mode
13               Fold Generator                       StratifiedKFold
14                  Fold Number                                    10
15                     CPU Jobs                                    -1
16                      Use GPU                                 False
17               Log Experiment                                 False
18              Experiment Name                      clf-default-name
19                          USI                                  8630
[2024-07-07T05:32:17.112+0000] {logging_mixin.py:188} WARNING - Processing:   0%|          | 0/61 [00:00<?, ?it/s]
[2024-07-07T05:32:17.311+0000] {logging_mixin.py:188} WARNING - Processing:   5%|4         | 3/61 [00:00<00:03, 16.44it/s]
[2024-07-07T05:32:32.062+0000] {job.py:213} ERROR - Job heartbeat got an exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
           ^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
          ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: could not translate host name "postgres" to address: Temporary failure in name resolution


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/jobs/job.py", line 187, in heartbeat
    self._merge_from(Job._fetch_from_db(self, session))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/api_internal/internal_api_call.py", line 114, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/jobs/job.py", line 308, in _fetch_from_db
    session.merge(job)
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 3056, in merge
    return self._merge(
           ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 3136, in _merge
    merged = self.get(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 2853, in get
    return self._get_impl(
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 2975, in _get_impl
    return db_load_fn(
           ^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/loading.py", line 530, in load_on_pk_identity
    session.execute(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 1716, in execute
    conn = self._connection_for_bind(bind)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 1555, in _connection_for_bind
    return self._transaction._connection_for_bind(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/orm/session.py", line 750, in _connection_for_bind
    conn = bind.connect()
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/future/engine.py", line 412, in connect
    return super(Engine, self).connect()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3325, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3404, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3374, in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 2208, in _handle_dbapi_exception_noconnection
    util.raise_(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
           ^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
          ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 690, in __connect
    with util.safe_reraise():
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2024-07-07T05:32:32.439+0000] {job.py:221} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
[2024-07-07T05:32:37.726+0000] {logging_mixin.py:188} WARNING - Processing:   7%|6         | 4/61 [00:20<00:03, 16.44it/s]
[2024-07-07T05:37:27.904+0000] {logging_mixin.py:188} WARNING - Processing:   8%|8         | 5/61 [05:10<1:10:43, 75.78s/it]
[2024-07-07T05:37:28.421+0000] {logging_mixin.py:188} WARNING - Processing:  11%|#1        | 7/61 [05:11<40:17, 44.77s/it]  
[2024-07-07T05:38:18.479+0000] {logging_mixin.py:188} WARNING - Processing:  15%|#4        | 9/61 [06:01<32:28, 37.47s/it]
[2024-07-07T05:38:19.125+0000] {logging_mixin.py:188} WARNING - Processing:  18%|#8        | 11/61 [06:02<20:31, 24.63s/it]
[2024-07-07T05:38:24.376+0000] {logging_mixin.py:188} WARNING - Processing:  21%|##1       | 13/61 [06:07<13:53, 17.36s/it]
[2024-07-07T05:38:24.783+0000] {logging_mixin.py:188} WARNING - Processing:  25%|##4       | 15/61 [06:07<09:05, 11.86s/it]
[2024-07-07T05:38:39.008+0000] {logging_mixin.py:188} WARNING - Processing:  28%|##7       | 17/61 [06:21<07:36, 10.36s/it]
[2024-07-07T05:38:39.770+0000] {logging_mixin.py:188} WARNING - Processing:  31%|###1      | 19/61 [06:22<05:05,  7.27s/it]
[2024-07-07T05:39:55.138+0000] {logging_mixin.py:188} WARNING - Processing:  34%|###4      | 21/61 [07:38<11:04, 16.60s/it]
[2024-07-07T05:39:55.556+0000] {logging_mixin.py:188} WARNING - Processing:  38%|###7      | 23/61 [07:38<07:21, 11.61s/it]
[2024-07-07T05:40:02.520+0000] {logging_mixin.py:188} WARNING - Processing:  41%|####      | 25/61 [07:45<05:29,  9.14s/it]
[2024-07-07T05:40:03.057+0000] {logging_mixin.py:188} WARNING - Processing:  44%|####4     | 27/61 [07:45<03:39,  6.46s/it]
[2024-07-07T05:40:34.460+0000] {logging_mixin.py:188} WARNING - Processing:  48%|####7     | 29/61 [08:17<04:55,  9.25s/it]
[2024-07-07T05:40:35.661+0000] {logging_mixin.py:188} WARNING - Processing:  51%|#####     | 31/61 [08:18<03:19,  6.64s/it]
[2024-07-07T05:40:42.125+0000] {logging_mixin.py:188} WARNING - Processing:  54%|#####4    | 33/61 [08:25<02:37,  5.62s/it]
[2024-07-07T05:40:42.498+0000] {logging_mixin.py:188} WARNING - Processing:  57%|#####7    | 35/61 [08:25<01:43,  3.99s/it]
[2024-07-07T05:41:21.006+0000] {logging_mixin.py:188} WARNING - Processing:  61%|######    | 37/61 [09:03<03:25,  8.57s/it]
[2024-07-07T05:41:21.740+0000] {logging_mixin.py:188} WARNING - Processing:  64%|######3   | 39/61 [09:04<02:14,  6.11s/it]
